samples/app-complete-auth/nodejs/README.md

```diff
@@ -112,7 +112,7 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
    - Select Configuration section.
    - Under configuration -> Add OAuth connection string.
    - Provide connection Name : for eg `ssoconnection`
-   - Select service provider ad `Azure Active Directory V2`
+   - Select service provider ad `Microsoft Entra ID V2`
```

Comment:
Need to Undo this change, Keep Azure Active Directory .

Reply:
fixed